### DISCUSSION TITLE
Is there a way to debug the mail job that run by the sidekiq?

### DISCUSSION THREAD

**Discussion by Naomi1900** [chatwoot/chatwoot#7823] [Self hosted]
Is there a way to debug the mail job that run by the sidekiq?

**Comment by jtayrone** [comment]
You can view the web logs with:

`cwctl --logs web`

Or you can test if the SMTP service is properly configured by following the documentation to [debug SMTP email errors.](https://www.chatwoot.com/docs/self-hosted/faq#how-to-debug-smtp-email-errors-in-chatwoot/)
